solvers/SimpleSOL/ParticleSystems/neutral_particles.hpp

```diff
@@ -346,8 +358,10 @@ class NeutralParticleSystem {
     NESOASSERT(this->field_project != nullptr,
                "Field project object is null. Was setup_project called?");
 
-    std::vector<Sym<REAL>> syms = {Sym<REAL>("SOURCE_DENSITY")};
-    std::vector<int> components = {0};
+    std::vector<Sym<REAL>> syms = {
+        Sym<REAL>("SOURCE_DENSITY"), Sym<REAL>("SOURCE_MOMENTUM"),
+        Sym<REAL>("SOURCE_MOMENTUM"), Sym<REAL>("SOURCE_ENERGY")};
```

Comment:
Why is `SOURCE_MOMENTUM` in here twice?

Reply:
The args to project are vectors of Syms and indices ("components) that get paired up. Momentum is a vector quantity, so it can have index>0
The quantities to project are therefore (density,0); (momentum,0); (momentum,1); (energy,0)